### ISSUE TITLE
Credo.Check.Readability.StringSigils should not warn on multiline string

### ISSUE THREAD

**Issue by chulkilee** [rrrene/credo#611]
### Environment

* Credo version (`mix credo -v`): 1.0.0
* Erlang/Elixir version (`elixir -v`): 1.7.4
* Operating system: mac

### What were you trying to do?

Use multiline string to wrap JSON string

```elixir
"""
{
  "hello": "world"
  "foo": "bar"
}
"""
|> Jason.parse()
```

### Expected outcome

No error from Credo.Check.Readability.StringSigils since it's already readable :)

Note that `~s` cannot be used in that case since it does not remove leading spaces (like `"""`).

### Actual outcome

Credo.Check.Readability.StringSigils is not happy with it

> More than 3 quotes found inside string literal, consider using a sigil instead.

**Comment by chulkilee** [comment]
I'm curious whether that info (string is multiline or not) is available to Credo.

**Comment by rrrene** [comment]
@chulkilee You are right, this should not warn for multi-line strings ("heredocs"). I'm not sure why it does, since I was not able to reproduce this.

Could you provide a file as a test-case? (oh, and sorry, I completely missed your issue 😔)

**Comment by chulkilee** [comment]
Updated the example - it happens when I pass such string into pipe.

**Comment by rrrene** [comment]
Got it. Thanks! Will work on a fix :+1:

**Comment by rrrene** [comment]
@chulkilee 
Thanks for reporting this 😀  It is now fixed on `master`.

You can try this by setting the Credo dep to

```elixir
{:credo, github: "rrrene/credo"}
```

Please report back if your issue is solved! 👍
